sdks/python/apache_beam/io/azure/integration_test/create_certificate.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
e.g. We still need Apache License header here. And better move the raw .pem files here and write them to local at test run time as part of this script.

Reply:
I deleted `create_certificate.sh` and moved the cert creation code to `azure_integration_test.sh`, then I deleted `cert.pem` and `key.pem` to avoid the license header issues.

Generating the certificate on every test run adds just a second or two to the total test time.

I hope that solution is acceptable, but happy to change it if not.